packages/subgraph/src/helpers/conviction.ts

```diff
@@ -43,10 +44,15 @@ export function loadConvictionConfig(orgAddress: Address, appAddress: Address):
   if (stakeTokenId) {
     convictionConfig.stakeToken = stakeToken.toHexString()
 
-    // Set token for org
+    // Set token for org and vice versa
```

Comment:
I think this is all right for now. Regardless I was considering if having the initialization of the token here was the best place.

Reply:
On second thought, maybe we can set the token in the [`setUpHoneyLiquidity()`](https://github.com/PJColombo/gardens/blob/f63f8320afb47f719698f248f3d56d555208ad39/packages/subgraph/src/helpers/pair.ts#L38) function, where we have access to the dao address. What do you think ?

Reply:
Yeah i think this makes sense. And we can be rest assured the org will have the associated token already.

One important thing that we need to take into account is the fact that for BYOT gardens, the token that is going to be paired with HNY will be the `org.wrappableToken` so we should check first if the org has a `wrappableToken` associated and use that one. In the case it doesn't fallback to the `org.token`